### ISSUE TITLE
vtkwindow is null

### ISSUE THREAD

**Issue by zengxiuhao** [dearman/activizdotnet#12]
iI put the render control in the tab but it doesn't show up.
I loaded the 3d data into memory.
but vs report  vtkrenderwindow is null.
if i frist show the rendercontrol in the mianform.  it runs no error.
i think the vtkwindow do Not created in the constructor。

how can i do to resolve problem

**Comment by dearman** [comment]
I don't believe I have a build environment setup for this project anymore.  Maybe in a couple weeks I can get it going again and take a look at your code.

**Comment by zengxiuhao** [comment]
i create a project  test；
the project contain   a tabcontrol and zhe tabcontrol have 4 tab.
the second tab contain a rendercontrol.
defalut. the frist tab is select.
when i  start application.
frist  tab is selected and other tab is not select.
and i click a button .to check the vtkrenderwindow of rendercontrol  is null？
![微信截图_20241128105949](https://github.com/user-attachments/assets/8cd4b332-f9ea-44a0-a3da-2db30141201f)

and my code :

![微信截图_20241128110046](https://github.com/user-attachments/assets/9ec80ae0-e1d8-4724-9f61-f1ec7ddea06c)

when i click the button ,the messagebox is show render window is null.

why i ask this quetion,because ,i need to load data to render control in back. i click the tab ,it can show
